src/w1thermsensor/cli.py

```diff
@@ -59,27 +59,49 @@ def cli():
 @click.option(
     "-j", "--json", "as_json", flag_value=True, help="Output result in JSON format"
 )
-def ls(types, as_json):  # pylint: disable=invalid-name
+@click.option(
+    "-p",
```

Comment:
Why is it `-p` ?

Reply:
just copied what was used for the 'all' command

Reply:
let me know if you want me to remove it

Reply:
I see, my mistake - this was probably introduced when I renamed `precision` to `resolution`. 
I'll change that in master :)